packages/website/src/components/linter/createLinter.ts

```diff
@@ -160,7 +167,37 @@ export function createLinter(
       console.log('[Editor] Updating', fileName, compilerOptions);
       parser.updateConfig(compilerOptions);
     } catch (e) {
-      console.error(e);
+      if (e instanceof Error) {
+        const error = {
+          group: 'Typescript',
+          items: e.message
+            .trim()
+            .split('\n')
+            .map((message: string) => {
+              return {
+                fixer: undefined,
+                location: '',
+                message,
+                severity: 8,
+                suggestions: [],
+              };
+            }),
+          uri: undefined,
+        };
+
+        onMarkersChange(prev => {
+          const activeTabErrors = Object.fromEntries(
+            prev.tsconfig.map(error => [error.group, error]),
+          );
+
+          activeTabErrors.Typescript = error;
```

Comment:
Here and in other places that refer to `Typescript`:

```suggestion
          activeTabErrors.TypeScript = error;
```